distributionviewer/core/static/js/app/components/layouts/main-layout.js

```diff
@@ -11,19 +11,27 @@ export default function(props) {
       <header className="primary-header">
         <div>
           <h1>
-            <Link className="primary-header-link" to="/">Firefox Distribution Viewer</Link>
+            <Link className="primary-header-link" to={'/?pop=' + props.whitelistedPopulations}>Firefox Distribution Viewer</Link>
```

Comment:
I didn't see above...what happens if `props.whitelistedPopulations` is undefined?